docs/libraries/http/reference/data-types.md

```diff
@@ -126,6 +185,12 @@ The request conflicts with the current state of the server.
 model TypeSpec.Http.ConflictResponse
 ```
 
+#### Properties
+
+| Name       | Type           | Description      |
+| ---------- | -------------- | ---------------- |
+| statusCode | `(number) 409` | The status code. |
```

Comment:
This seems a bit weird. Perhaps it would make more sense to be "Literal (number) = 409" or something? Not for the PR (unless that's super easy) but maybe a follow-up issue?